horace_core/configuration/@hor_config/hor_config.m

```diff
@@ -186,6 +197,28 @@
             end
             config_store.instance().store_config(this,'mem_chunk_size',val);
         end
+
+        function this = set.pixel_page_size(this, val)
+            MIN_RECOMMENDED_PG_SIZE = 100e6;
+            if val < PixelData.DATA_POINT_SIZE*PixelData.DEFAULT_NUM_PIX_FIELDS
```

Comment:
you compare against `PixelData.DATA_POINT_SIZE*PixelData.DEFAULT_NUM_PIX_FIELDS` but the error message says `72`